lib/SILGen/SILGenPattern.cpp

```diff
@@ -1812,8 +1812,7 @@ CaseBlocks::CaseBlocks(
                                SGF.F.getResilienceExpansion());
   }
   if (canAssumeExhaustive) {
-    // Check that Sema didn't let any cases slip through. (This can happen
-    // with @_downgrade_exhaustivity_check.)
+    // Check that Sema didn't let any cases slip through.
```

Comment:
I think this can't happen anymore, so maybe we can make this an assertion instead.